### ISSUE TITLE
Uploaded images are not showing up

### ISSUE THREAD

**Issue by kajalshah** [Requarks/wiki#216]
### Actual behavior
When I upload image to be inserted in the wiki page, i dont see the newly uploaded image.

### Expected behavior
Any uploaded image should be listed so that it can be inserted in the page.

### Steps to reproduce the behavior
1. Create new Wiki page or either edit existing wiki page
2. Click on insert image icon
3. Upload new image from the computer
4. Once image uploading process is done, I cannot see the image uploaded.
If i re-upload image i see error but not the image. I closed the popup and returned to upload image and got error shown in the image.
![issue-image upload](https://user-images.githubusercontent.com/6513000/29575913-9bc75580-8783-11e7-86a8-7a972d5e3d69.png)

**Comment by NGPixel** [comment]
If you restart your wiki, do you still have the same issue?

**Comment by kajalshah** [comment]
yes

**Comment by kajalshah** [comment]
any updates?

**Comment by Bamieh** [comment]
@kajalshah check the permissions of the folder its being uploaded to

**Comment by kajalshah** [comment]
@Bamieh can you share with me how can i check permissions?

**Comment by Bamieh** [comment]
@kajalshah which OS do you use?

**Comment by kajalshah** [comment]
@Bamieh windows 8

**Comment by Bamieh** [comment]
@kajright check [this link](https://www.isunshare.com/windows-8/set-folder-permissions-in-windows-8-8.1-computer.html) or just google
> how to change permissions of a folder in windows. try to make the file public at first, then reduce permissions as needed if it works.

**Comment by kajalshah** [comment]
@Bamieh thanks for sharing this link.
i checked the permission and it is fine. I was able to upload images earlier and suddenly after the image is uploaded on wiki page, i dont see them. refer the screen i attached with the issue above.

**Comment by NGPixel** [comment]
Does the image exists on disk? (In your repo folder > uploads)

**Comment by kajalshah** [comment]
yes it exists on disk

**Comment by NGPixel** [comment]
Seems like the worker is unable to process your image then. Have you tried with other images? Do you get the same result?

**Comment by kajalshah** [comment]
i have tried with different images. existing images and even with new images which were not uploaded.
all the time it completes uploading and then i dont see any image uploaded on the screen i showed you.
If i cant see image i uploaded how can i select it and insert it.

**Comment by kajalshah** [comment]
can i get any solution on this issue?

**Comment by NGPixel** [comment]
You can't see them in the list because it was unable to generate the thumbnail.
Error during parsing / thumbnail generation = not listed in the UI.

Yet you still get a "File already exists" error because the file does exist on disk.

Do you have any error in the logs?

**Comment by NGPixel** [comment]
No I can't get your error log, this isn't a service, you host it yourself. Logs are located in the `/logs` folder where you installed Wiki.js.

If there was a quick solution, it would be fixed already. I can't reproduce your issue and this isn't a widespread problem, so it takes time to find the cause.

**Comment by ixdarchitects** [comment]
I had the same issue. Very annoying!

**Comment by ikarulus** [comment]
I experience this issue with docker. After the initial install, first upload worked. All new files triggered the error that the first uploaded file already exists althought the new files all have other names. Also the two first files are not showing up in the menu.
When I access the image folder manually, I only get the error that it contains errors.

**Comment by n00bsys0p** [comment]
I've had exactly this behaviour while using the Docker container, and found a resolution. Mounting the logs directory was what got me on the right path.

The problem was caused by the remote repository first of all not being accessible as I hadn't mounted the Git SSH key correctly, and then once I resolved that another issue:... Because the remote repository was empty, I needed to log into the container and manually push the repository using the mounted Git SSH key. This manifested as an error stating that WikiJS couldn't find the remote master branch.

The former is just PICNIC, but the latter looks more like a bug, as surely we expect the repository to be empty under most circumstances when spinning up a new instance.

All working great now I've taken the above steps though.

**Comment by tanyeun** [comment]
Thanks to n00bsys0p 's hint, I was able to fix this as well.
To be more specific, I was able to make it work with http but not ssh. Here's how:

Edit your config.yml file:
(Normally shown as "git clone http://[username]@[repo address]" on your git server)
url: http://[repo address]
branch: master
auth:
    type: basic
    username: [username]
    password: [password]

**Comment by sindunuragarp** [comment]
A problem related to this is #327 which is a bug in the thumbnail generation, causing existing images to not get listed
